src/includes/getting-started-primer/flutter.mdx

```diff
@@ -1,7 +1,18 @@
 <Note>
 
-_Sentry's Flutter SDK enables capturing sessions for Release Health, offline caching as well as reporting messages and errors._
-
-_Sentry's Flutter SDK depends on the [Dart SDK](/platforms/dart/) and includes support to native crashes through Sentry's native SDKs (Android and iOS), It'll capture errors in the native layer, including Java, Kotlin, C and C++ code for `Android` and ObjC, Swift and C for `iOS`._
+_Sentry's Flutter SDK enables automatic reporting of errors, messages, and exceptions. The SDK is available on GitHub [`sentry-dart`](https://github.com/getsentry/sentry-dart/)._
 
 </Note>
+
+Features:
+
+- Automatic native crash error tracking (using both [Android](/platforms/android/) and [iOS](/platforms/apple/)), including
+    - Java, Kotlin, C, and C++ code for Android
+    - ObjC, Swift, and C for iOS
+- Offline storage of events
+- Events [enriched](/platforms/flutter/enriching-events/context/) with device data
+- Breadcrumbs automatically captured via the Native SDKs.
```

Comment:
but not only via the Native SDKs, we also have automatic breadcrumbs for Flutter https://sentry-docs-git-update-flutter-docs-improvements.sentry.dev/platforms/flutter/enriching-events/breadcrumbs/#automatic-breadcrumbs and `http` via the Dart SDK

Reply:
This was old content then (pulled from Advanced Usage); glad we can get it corrected.

I'll add "Breadcrumbs automatically captured by the [Flutter SDK](/platforms/flutter/enriching-events/breadcrumbs/#automatic-breadcrumbs) as well as via the Native SDKs."